package.json

```diff
@@ -167,14 +172,18 @@
     "pull-stream": "^3.6.8",
     "pull-stream-to-stream": "^1.3.4",
     "pull-zip": "^2.0.1",
+    "pump": "^3.0.0",
     "read-pkg-up": "^4.0.0",
     "readable-stream": "2.3.6",
     "receptacle": "^1.3.2",
+    "safe-buffer": "^5.1.2",
```

Comment:
Is this needed? Only one use in the code, and we don't use it anywhere else that we're using buffers.